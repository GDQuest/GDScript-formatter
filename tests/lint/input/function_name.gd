#region Good

func good_func():
	pass

func _private_good_func():
	pass

#endregion

#region Bad

func BadFunc():
	pass

func _PrivateBadFunc():
	pass

func Bad_Func():
	pass

func bad_Func():
	pass

#endregion