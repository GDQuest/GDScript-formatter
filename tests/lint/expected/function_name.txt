tests/lint/input/function_name.gd:13:function-name:error: Function name 'BadFunc' should be in snake_case, _private_snake_case format
tests/lint/input/function_name.gd:16:function-name:error: Function name '_PrivateBadFunc' should be in snake_case, _private_snake_case format
tests/lint/input/function_name.gd:19:function-name:error: Function name 'Bad_Func' should be in snake_case, _private_snake_case format
tests/lint/input/function_name.gd:22:function-name:error: Function name 'bad_Func' should be in snake_case, _private_snake_case format